core/src/UserSettingsDialog.html

```diff
@@ -379,114 +447,238 @@
     }
+    /*micro frontend and iframe wrappers inside the right-side dialog body*/
+    .mf-wrapper,
+    .iframe-wrapper {
+      overflow: auto;
+      height: 100%;
+    }
 
-    .lui-usersettings-content {
-      left: 0;
+    .lui-usersettings-dialog {
+      max-width: 100%;
     }
 
-    .lui-usersettings-content-header {
+    .lui-usersettings-dialog-size {
+      height: 80vh;
+    }
+
+    .lui-usersettings-content {
       left: 0;
+      height: 72vh;
+    }
+    .lui-fd-side__nav-wrapper {
+      height: 67vh;
     }
 
     .lui-usersettings-left-nav {
       width: 100%;
       transition: 0.5s;
     }
 
-    .fd-nested-list__title {
-      padding-right: 0;
+    .fd-nested-list__link {
+      height: 5rem;
+    }
+
+    .fd-nested-list__icon img {
+      max-width: 18px;
+      max-height: 18px;
+    }
+
+    .lui-usersettings-list-item__info-container {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: flex;
+      align-items: center;
+    }
+
+    .lui-usersettings-list-item__title-container {
+      display: flex;
+      flex-direction: column;
+      flex: auto;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    .lui-usersettings-list-item__title {
+      font-size: 0.875rem;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      flex: 1 1 auto;
+    }
+
+    .lui-usersettings-list-item__subtitle {
+      font-size: 0.875rem;
+      color: #6a6d70;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      flex: 1 1 auto;
+      padding-top: 0.875rem;
+    }
+
+    .iframeUserSettingsCtn {
+      position: relative;
+      width: 100%;
+      height: 100%;
+    }
+
+    .iframeUserSettingsCtn :global(iframe) {
+      width: 100%;
+      height: 100%;
+      border: none;
+    }
+
+    .iframeUserSettingsCtn {
+      overflow: visible;
+      -webkit-overflow-scrolling: touch;
+    }
+
+    .lui-usersettings-content-header__back-btn {
+      display: none;
+      float: left;
+      width: 0;
     }
 
-    .lui-usersettings-content-header__backBtn {
-      display: block;
-      width: 35%;
-      margin: 0;
-      position: absolute;
-      top: 50%;
-      -ms-transform: translateY(-50%);
-      transform: translateY(-50%);
+    .lui-usersettings-content-header {
+      position: -webkit-sticky;
+      position: sticky;
+      top: 0;
+      width: 100%;
+      height: 50px;
     }
 
     .lui-usersettings-content-header__title {
-      width: 65%;
+      width: 100%;
+      font-size: 16px;
+      font-weight: bold;
+      text-align: center;
+      line-height: 50px;
+    }
+
+    @media (max-width: 600px) {
+      :global(.usersettings-leftNavVisible) .lui-usersettings-left-nav {
+        width: 0;
+      }
+      /*micro frontend and iframe wrappers inside the right-side dialog body*/
+      .mf-wrapper,
+      .iframe-wrapper {
+        overflow: auto;
```

Comment:
I have a double scrollbar on iframe mf.
When I remove this `overflow:auto`, I don't have it.
Does it make sense? Can't see any side-effect.